### PR TITLE
DP-3588: Removing 'filterBy function that duplicates build-in 'filter'

### PR DESCRIPTION
**Pull request by KlaudiaJ**

### CODE REVIEW

File: lib/common.js [hunk 1]
```diff
@@ -71,22 +71,6 @@ module.exports.initializeFile = function(filepath, data) {
 };
 
 /**
- * @description filter and array using the condition parameter
- * @param array
- * @param condition
- * @returns {Array}
- */
-module.exports.filterBy = function (array, condition) {
```
Comment (jrozestw) [inline lib/common.js]: did you check if this function is used anywhere?

File: test/lib_commonTests.js [hunk 2]
```diff
@@ -63,24 +63,6 @@ describe(fileToTest, function(){
         done();
 
     });
-    it('Shall Filter the objects comply with condition >', function(done){
```
Comment (jrozestw) [inline test/lib_commonTests.js]: did you run those tests for function 'filter' to check if behavior is the same?

Comment (KlaudiaJ) [inline test/lib_commonTests.js]: In fact no. I checked it manually and by documentation.

File: lib/common.js (hunk 1 above)
Comment (KlaudiaJ) [inline lib/common.js]: yes, I searched whole project but it wasn't

**Pull request closed without merging**